hdfs_datanode/tests/conftest.py

```diff
@@ -4,11 +4,34 @@
 
 import json
 import os
+from copy import deepcopy
 
 import pytest
 from mock import patch
 
-from .common import HERE, TEST_PASSWORD, TEST_USERNAME
+from datadog_checks.dev import docker_run
+from datadog_checks.hdfs_datanode import HDFSDataNode
+
+from .common import HERE, INSTANCE_INTEGRATION, TEST_PASSWORD, TEST_USERNAME
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    with docker_run(
+        compose_file=os.path.join(HERE, "compose", "docker-compose.yaml"),
+        log_patterns='Got finalize command for block pool',
+    ):
+        yield INSTANCE_INTEGRATION
```

Comment:
I think you can remove the value from the yield, it's duplicated in the instance fixture otherwise.